gammapy/irf/psf/core.py

```diff
@@ -250,6 +216,42 @@ def plot_containment_radius(self, ax=None, fraction=0.68,  add_cbar=True, **kwar
 
         return ax
 
+    def plot_psf_vs_rad(self, ax=None, offset=[0] * u.deg, energy_true=[0.1, 1, 10] * u.TeV, **kwargs):
```

Comment:
Could it be possible to add a `offset = Angle(offset)` to allow strings?

Reply:
Done

Reply:
BTW: this is something we should check for uniformity in Gammapy. In general I can see how it is convenient, however it adds a lot of code, where we convert to `Quantity` explicitly and also need to document that strings can be passed as well...I'll also add it to the agenda for the next dev meeting.